xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/test/java/org/xwiki/rendering/internal/macro/IncludeMacroParametersTest.java

```diff
@@ -41,4 +41,15 @@ public void setPage()
 
         assertSame(EntityType.PAGE, parameters.getType());
     }
+    
+    @Unstable
```

Comment:
I made changes to the existing code that's why I added `@Unstable`. So It can appear in the backward-compatibility reports.
I read here a few days ago https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices.
I was not really sure to put them for test methods. I am again looking for more information. Thank you!

Reply:
Yes you can read about it here: https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HBackwardCompatibility

> I made changes to the existing code that's why I added @Unstable.

But why would making changes to existing code require to add an `@Unstable` annotation? (I know the answer ofc but I'm trying to make it you find it :)).

Reply:
Thanks for the link. What I understand is @Unstable is used to Stabilize the public APIs. So When a user uses a class or method annotated with @Unstable he will not certain about stabilization and he can use it at his own risk. When that method or class is stable to use the @Unstable annotation will be removed. It helps in maintaining the backward-compatibility.
For test methods, we do not add this annotation because Tests are added for developers to test their code and not for the user to use them.